setup/kubernetes/google.md

```diff
@@ -78,6 +78,18 @@ gcloud beta container --project "$PROJECT_ID" \
     --max-nodes "8"
 ```
 
+## Access Control
+
+GKE allows for integration between Identity Access and Management (IAM) and Kubernetes'
+native Role-Based Access Control (RBAC) mechanism to authorize user actions in the
+cluster. IAM configuration is primarily applied at the Google Cloud Project level,
+and all clusters within that Project. Kubernetes RBAC configuration applies to each
+individual cluster, allowing for fine-grained authorization down to the namespace
+level.
+
```

Comment:
future state - we could reference the diagrams [in this article](https://medium.com/uptime-99/making-sense-of-kubernetes-rbac-and-iam-roles-on-gke-914131b01922#CommonIAMandRBACRoles) to clarify the differences between GKE & K8s roles.